lib/eibflatpak.py

```diff
@@ -883,103 +911,61 @@ def _do_pull(self, repo, remote, options):
         finally:
             progress.finish()
 
-    def pull(self, commit_only=False, cache_repo_path=None):
-        """Pull all refs to install
+    def seed(self, cache_repo_path, refs):
+        """Pull commits from a cache repo to the installation repo
 
-        Use OSTree to pull all the needed refs to a repository. If
-        commit_only is True, the commit checksums are pulled without
-        creating repository refs. If cache_repo_path points to an ostree
-        repo, it will be used as a local object cache.
+        This is used during install to get as many commit objects as
+        possible into the installation repo ahead of time. It would be
+        better if you could specify the OSTree pull localcache-repos
```

Comment:
boooo

Reply:
I opened https://github.com/flatpak/flatpak/issues/1284 a while back. I think with the transaction code it would be easier to narrow that to just adding a `flatpak_transaction_set_localcache_repos` API. That feature is already used internally when pulling to a user repo and using the system repo as a cache (I think).